src/main/java/marquez/common/models/DatasourceName.java

```diff
@@ -0,0 +1,14 @@
+package marquez.common.models;
```

Comment:
minor: add license

Reply:
Please see PRs #394, #387, and #384, where this is fixed.